src/internal/protoutil/protoutil.go

```diff
@@ -0,0 +1,48 @@
+// Package protoutil contains some utilities for interacting with protocol buffer objects.
```

Comment:
I like this.

You might be able to implement `MustTimestampFromPointer` in terms of `MustTimestamp`.  That would make it more difficult for their (currently identical) error messages to get out of sync.

Reply:
So in part 3, neither of these can error anymore, so I'm fine leaving this as is for now.  With the new API, you get <timestamp>.AsTime() and timestamppb.New(<go time>), and neither can error.